VHSchan support

MrPurple666 opened the pull request.

MrPurple666 committed:
vhschan initial commit

Commit:
vhschan initial commit

K1rakishou commented on Kuroba/app/src/test/java/com/github/k1rakishou/chan/core/site/parser/ReplyParserTest.kt:
```diff
@@ -94,6 +95,18 @@ class ReplyParserTest {
+  fun `test extract quotes from vhschan_org comment`() {
+    whenever(siteManager.bySiteDescriptor(any())).thenReturn(Vhschan())
+
+    val replies = replyParser.extractCommentReplies(SiteDescriptor.create("vhschan.org>
```
It seems like something wrong went here.

MrPurple666 committed:
fix ">"

> to ), VICHAN_PARSER_TEST_COMMENT)

MrPurple666 committed:
fix ">"

> to ), VICHAN_PARSER_TEST_COMMENT)

K1rakishou committed:
Change endpoints, remove not working test.

K1rakishou committed:
Change endpoints, remove not working test.

K1rakishou commented:
The favicon doesn't work (returns 404). Some endpoints were incorrect (apparently) so I changed them. Some thumbnails do not show up (404). If this is fine then I can merge it otherwise you can try to fix it.

MrPurple666 committed:
Fixes and renames

fixed favicon.ico, fixed name writing issues (Vhschan -> vhschan), board adjustment and added recent boards

MrPurple666 commented:
> The favicon doesn't work (returns 404). Some endpoints were incorrect (apparently) so I changed them. Some thumbnails do not show up (404). If this is fine then I can merge it otherwise you can try to fix it.

I fixed favicon, renamed some things to avoid problems as many were with a upercase "V" so everything is in lowercase, boards name and description have been fixed! Probably the issues with 404 were also fixed with these changes.

MrPurple666 commented:
Sorry for force push, my git setup was wrong and the commits "were not" my own (bcz wrong email). It will not happen again.

K1rakishou committed:
Merge remote-tracking branch 'MrPurple666/develop' into MrPurple666_develop

# Conflicts:
#	Kuroba/app/src/main/java/com/github/k1rakishou/chan/core/site/SiteRegistry.kt

K1rakishou committed:
Rename the class to be in CamelCase.

K1rakishou committed:
Fix thumbnails.

K1rakishou reviewed (approved):
LGTM
